docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -442,10 +302,129 @@ This file is picked up when the server starts as indicated by the following AUDI
+                            "portNumber": 50000,
+                            "user": "db2inst1",
+                            "password": "********",
+                            "databaseName": "FHIRDB",
+                            "currentSchema": "FHIR1",
+                            "driverType": 4
+                        }
+                    }
+                }
+            }
+        }
+    }
+    ```
+
+    The `connectionProperties` property is a set of driver-specific properties needed to connect to an instance of that database type. For a Db2-related datasource definition, any bean property supported by the `DB2XADataSource` class can be specified within the `connectionProperties` property group. For a discussion of the specific properties that can be used to configure a `DB2XADataSource` instance, see the [Db2 Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSEPGG_11.1.0/com.ibm.db2.luw.apdv.java.doc/src/tpc/imjcc_rjvdsprp.html).
+
+    For a Derby-related datasource definition, any bean property supported by the `EmbeddedXADataSource` class can be specified within the `connectionProperties` property group. For more information about the properties supported by the `EmbeddedXADataSource` class, and its super classes, see the [Apache Derby documentation](https://db.apache.org/derby/docs/10.13/publishedapi/org/apache/derby/jdbc/EmbeddedXADataSource.html).
+
+
+When the proxy datasource is obtained via a JNDI lookup by the JDBC persistence layer, and then its “getConnection()” method is called, the proxy datasource will use the current tenant-id and datastore-id to retrieve the corresponding configuration properties from the `fhir-server-config.json` file. The result will be the instantiation (or a successful cache lookup) of the appropriate XADataSource implementation class (according to the `type` field in the datastore configuration) and then a new connection will be obtained from it.
+
+The proxy datasource relies on the presence of the tenant-id and datastore-id information within the thread-local `FHIRRequestContext` information. In a simple configuration of the FHIR server that involves the use of the JDBC persistence layer implementation, the `FHIRRequestContext` information is obtained via request headers from the incoming REST API request. The tenant-id is obtained from the `X-FHIR-TENANT-ID` request header and the datastore-id is obtained from the `X-FHIR-DSID` request header<sup id="a3">[3](#f3)</sup>.
+
+#### 3.3.1.3 Supported databases
+##### 3.3.1.3.1 Embedded Derby (default)
+If you are using the `ibmcom/ibm-fhir-server` docker image, you can ask the entrypoint script to create (bootstrap) the database and the schema and tables during startup by setting the `DB_BOOTSTRAP` environment variable to `true`.
+
+This database bootstrap step is only supported for Embedded Derby and will only bootstrap the default datastore of the default tenant (the default for requests with no tenant or datastore headers).
```

Comment:
Maybe mention again here how this is just to enable easy out-of-the-box setup, and not recommended for production use.

Reply:
Haha, we have that in like 5 other spots, but sure.
```suggestion
This database bootstrap step is only supported for Embedded Derby and will only bootstrap the default datastore of the default tenant (the default for requests with no tenant or datastore headers). Reminder: 
 the Embedded Derby support is designed to support simple getting started scenarios and is not recommended for production use.
```